Examples/Example-iOS/Source/GTMAppAuthExampleViewController.m

```diff
@@ -22,9 +22,12 @@
 
 @import AppAuth;
 @import GTMAppAuth;
-#if __has_include("GTMSessionFetcher/GTMSessionFetcher.h") // Cocoapods
-@import GTMSessionFetcher;
-#else // SPM
+// CocoaPods || SPM && GTMSessionFetcher >= 2.0
```

Comment:
@paulb777 any better ideas here?

Reply:
This seems to work for both GTMSessionFetcher 1.x and 2.x in .m files.

```
#if SWIFT_PACKAGE
@import GTMSessionFetcherCore;
#else
#import <GTMSessionFetcher/GTMSessionFetcher.h>
#import <GTMSessionFetcher/GTMSessionFetcherService.h>
#endif
```

Reply:
The `SWIFT_PACKAGE` macro is only available for internal headers and implementation of the SPM target, correct?

Reply:
I think what I have will work for now.